ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -922,11 +924,12 @@ private class PostgresLedgerDao(
       PageSize,
       (startI, endE) => {
         Source
-          .fromFuture(dbDispatcher.executeSql { implicit conn =>
-            SQL_GET_LEDGER_ENTRIES
-              .on("startInclusive" -> startI, "endExclusive" -> endE)
-              .as(EntryParser.*)
-              .map(toLedgerEntry)
+          .fromFuture(dbDispatcher.executeSql(s"load ledger entries [$startI, $endE[") {
```

Comment:
I usually see round parentheses to denote an exclusive boundary end. Is this a typo or a different notation I wasn't aware of?

Reply:
Turns out, the [notation](https://en.wikipedia.org/wiki/Interval_%28mathematics%29#Including_or_excluding_endpoints) is actually mixed with parenthesis and square brackets:
```
    (0, 5) = 1, 2, 3, 4
    (0, 5] = 1, 2, 3, 4, 5
    [0, 5) = 0, 1, 2, 3, 4
    [0, 5] = 0, 1, 2, 3, 4, 5
```